application/controllers/ParticipantController.php

```diff
@@ -199,6 +199,43 @@ public function allSchemesAction() {
         $this->view->globalQcAccess=$commonService->getConfig('qc_access');
     }
 
+    public function surveyAction() {
+        $this->_helper->layout()->activeMenu = 'view-reports';
+        $this->_helper->layout()->activeSubMenu = 'individual-reports';
+        if ($this->_hasParam('62cc026d0c2ab')) {
+            $db = Zend_Db_Table_Abstract::getDefaultAdapter();
+            $map_id = (int) base64_decode($this->_getParam('62cc026d0c2ab'));
```

Comment:
Out of curiosity, why do some developers like to name path params like this?

Reply:
I have no idea why that was there, but it looked intentional on a similar snippet so I followed the pattern. I guess the effort is to obscure the URL somehow, security through obscurity? Not a great principal if you ask me.